config/401-controller-service.yaml

```diff
@@ -31,3 +31,25 @@ spec:
     app.kubernetes.io/component: controller
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: pipelines-as-code
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pipelines-as-code-controller-metrics
+  namespace: pipelines-as-code
+  labels:
+    app.kubernetes.io/version: "devel"
+    app.kubernetes.io/part-of: pipelines-as-code
+    app: pipelines-as-code-controller
+spec:
+  ports:
+  - name: http-metrics
+    port: 9090
+    protocol: TCP
+    targetPort: 9090
+  selector:
```

Comment:
can't we include this port info in the existing controller service :thinking:

Reply:
I would prefer to have different service for listening and prometheus metrics. IMO this way it gives more maintainability, clarity and independency in handling the configuration, no?

Reply:
right but it leads to duplication of stuff

If you see even Pipeline have single service https://github.com/tektoncd/pipeline/blob/main/config/controller.yaml#L184-L192 for multiple things and with port it will be clarified 

WDYT ?

Reply:
I am ok with including in existing service as well. Not much difference for me

Reply:
@savitaashture done